datm/cime_config/config_component.xml

```diff
@@ -296,6 +296,15 @@
     <desc>starting year to loop data over</desc>
   </entry>
 
+  <entry id="DATM_YR_START_FILENAME">
+    <type>integer</type>
+    <valid_values></valid_values>
+    <default_value>UNSET</default_value>
```

Comment:
UNSET is not a valid integer, so this default may have to be zero or some other integer.